src/dapps/preview/emote-representation-with-blobs.ts

```diff
@@ -0,0 +1,7 @@
+import { EmoteRepresentationDefinition } from './emote-representation-definition'
+
+/** @alpha */
+export type EmoteRepresentationWithBlobs = Omit<
+  EmoteRepresentationDefinition,
+  'contents'
+> & { contents: { key: string; blob: any }[] }
```

Comment:
Should blob be treated as any? Can't it be typed as Blob?

Reply:
It breaks node, it was like that at first but then we had to change it to any

Reply:
here: https://github.com/decentraland/common-schemas/pull/108